mobile/src/main/java/org/openhab/habdroid/ui/WidgetListFragment.java

```diff
@@ -184,26 +203,30 @@ public void onItemLongClicked(final Widget widget) {
 
         if (widget.linkedPage() != null) {
             labels.add(getString(R.string.nfc_action_to_sitemap_page));
+            labels.add(getString(R.string.home_shortcut_pin_to_home));
         }
 
         if (!labels.isEmpty()) {
             final String[] labelArray = labels.toArray(new String[0]);
             new AlertDialog.Builder(getActivity())
                     .setTitle(R.string.nfc_dialog_title)
                     .setItems(labelArray, (dialog, which) -> {
-                        final Intent writeTagIntent;
                         if (which < commands.size()) {
-                            writeTagIntent = WriteTagActivity.createItemUpdateIntent(getActivity(),
-                                    widget.item().name(), commands.get(which),
-                                    labels.get(which), widget.item().label());
+                            startActivityForResult(WriteTagActivity.createItemUpdateIntent(
```

Comment:
While you're at it, you can make this call `startActivity` instead of `startActivityForResult` here and below. There's nothing listening for a result (no `onActivityResult` implementation) anyway.